pyproject.toml

```diff
@@ -68,20 +68,20 @@ coloredlogs = ">=10,<16"
 sanic = "^21.12.0"
 typing-extensions = ">=4.1.1,<5.0.0"
 Sanic-Cors = "^2.0.0"
-prompt-toolkit = "^3.0,<3.0.29"
+prompt-toolkit = ">=3,<4"
```

Comment:
For this update we need to double-check that the new micro works fine on `rasa` (testing CI and also`rasa interactive`) or if it breaks some functionality as it did in 3.0.29.